routesrv/eskipbytes.go

```diff
@@ -51,7 +59,8 @@ func (e *eskipBytes) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer span.Finish()
 
 	if data, initialized := e.bytes(); initialized {
-		w.Write(data)
+		w.Header().Add("Etag", e.etag)
```

Comment:
There is a race condition as etag could be modified after `e.bytes()` returns.
Lets drop `(e *eskipBytes) bytes()` and manually lock and unlock the mutex.